moto/backends.py

```diff
@@ -82,6 +82,7 @@
     "mediaconnect": ("mediaconnect", "mediaconnect_backends"),
     "mediapackage": ("mediapackage", "mediapackage_backends"),
     "mediastore": ("mediastore", "mediastore_backends"),
+    "mediastore-data": ("mediastore-data", "mediastoredata_backends"),
```

Comment:
The folder does not have a dash, so this should be:
```
"mediastore-data": ("mediastoredata", "mediastoredata_backends"),
```

Reply:
Sorted